fe/fe-core/pom.xml

```diff
@@ -482,12 +482,12 @@ under the License.
             <artifactId>grpc-stub</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.apache.doris</groupId>
-            <artifactId>hive-catalog-shade</artifactId>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
         </dependency>
         <dependency>
-           <groupId>org.apache.httpcomponents</groupId>
-           <artifactId>httpclient</artifactId>
```

Comment:
@CalvinKirs  The `doris-shade` release should contain this

Reply:
So I just make it consistent with master, right?

Reply:
I just remind myself:)It can also be done in[doris-shade](https://github.com/apache/doris-shade)if you are interested.